angular-chart.js

```diff
@@ -105,6 +114,23 @@
 
           if (usingExcanvas) window.G_vmlCanvasManager.initElement(elem[0]);
 
+          function aliasVar (fromName, toName) {
+            scope.$watch(fromName, function (newVal) {
+              if (typeof newVal == 'undefined') return;
```

Comment:
The `undefined` test is necessary, because angular `$watch` also triggers if the directive parameters are not specified (i.e., they are declared as `undefined`). E.g., if scope.chartData is `undefined`, we overwrite scope.data (which may have been previously assigned with the desired value). If we remove this line, practically all unit tests fail.